gaphor/ui/filemanager.py

```diff
@@ -177,11 +180,14 @@ def save(self, filename):
 
             self.filename = filename
             self.event_manager.handle(FileSaved(self, filename))
-        except Exception:
+        except Exception as e:
             error_handler(
-                message=gettext("Error while saving model to file {filename}").format(
+                message=gettext("Unable to save model “{filename}”.").format(
                     filename=filename
                 ),
+                secondary_message=gettext(
+                    "The model can not be stored at this location:\n{exc}"
```

Comment:
In the secondary messages, could we instead focus on the ways the user might be able to remedy the problem? For example: Please try to save a different location.